README.md

```diff
@@ -6,7 +6,7 @@ configured to include Substrate's [`pallet-contracts`](https://github.com/parity
 
 This repository is tracking Substrate's `master`.
 The last time it was synchronized with Substrate was up to
-[6b85535](https://github.com/paritytech/substrate/tree/6b8553511112afd5ae7e8e6877dc2f467850f155).
+[1c0d008](https://github.com/paritytech/substrate/tree/1c0d0083c2d3105c32b4fb7331f474ec2f0510dc).
```

Comment:
Acording to the lockfile the commit is: 3f70bcedf020d072e2f611a2448f2af46a4ae9b4

Reply:
Synced with [your last commit](https://github.com/paritytech/substrate/commit/021f71264133d1cf0db5bd3e0112dbf8dcecdd9d) on Substrate